Microsoft.Toolkit.Uwp.UI/Extensions/Markup/BitmapIconExtension.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Markup;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions.Markup
+{
+    /// <summary>
+    /// Custom <see cref="MarkupExtension"/> which can provide <see cref="BitmapIcon"/> values.
+    /// </summary>
+    [Bindable]
+    [MarkupExtensionReturnType(ReturnType = typeof(BitmapIcon))]
+    public sealed class BitmapIconExtension : MarkupExtension
+    {
+        /// <summary>
+        /// Gets or sets the <see cref="Uri"/> representing the image to display.
+        /// </summary>
+        public Uri Source { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to display the icon as monochrome.
+        /// </summary>
+        public bool ShowAsMonochrome { get; set; }
+
+        /// <inheritdoc/>
+        protected override object ProvideValue()
```

Comment:
>protected override object ProvideValue() [](start = 8, length = 40)

It can be a body-less method: 

```csharp
protected override object ProvideValue() => new BitmapIcon
{
    ShowAsMonochrome = ShowAsMonochrome,
    UriSource = Source
};
```

Reply:
Not really a fan of how the body expression looks in small methods returning an object like this, as at a glance it's easy to confuse the brackets under the method with the actual brackets for the method body, when in fact they belong to that constructor instead.
I'd prefer to leave this as is if that's alright, but let me know what you think! 😊

Reply:
@vgromfeld do you have any pros/cons around the styling of this one way or the other, I know this is a newer feature of C# compared to when the Toolkit started, so I don't think we have any explicit styling guidelines setup for this scenario.  Honestly though, I'm fine with either.

Reply:
The body-less generally allows us to remove all the unneeded characters and write more compact code. It might be less natural first but really helps to read the code faster once you get use to it 😀. 
VS2019 code style is recommending this approach. It definitively don't worth a full restyling of all the  toolkit code but should be more and more considered when writing new code.